ToolsTest/Models/VladyslavClassArticleTests/ArticleTestConstructor_xUnit.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tools.Models;
+using Xunit;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    public class ArticleTestConstructor_xUnit
+    {
+        [Fact]
+        public void CtorArticle_HasType_User()
+        {
+            //Arrange
+            Article _article = new Article();
+            User _user = new User("Tommy");
+            //Act
+            _article.Autor = _user;
+            //Assert
+            Assert.IsType<User>(_article.Autor);
+        }
+        [Fact]
+        public void CtorArticle_HasTitleAndText_TypeString()
+        {
+            //Arrange
+            Article _article = new Article();
+            //Act
+            _article.Title = "Title";
+            _article.Text = "Text";
+            //Assert
+            Assert.IsType<string>(_article.Title);
+            Assert.IsType<string>(_article.Text);
+        }
+        [Fact]
+        public void Ctor_AutorTitleText_NotNull()
+        {
+            //Arrange
+            Article _article = new Article();
+            //Act
+            _article.Autor = new User("Bill");
+            _article.Title = "Title";
+            _article.Text = "Text";
+            //Assert
+            Assert.NotNull(_article.Autor);
+            Assert.NotNull(_article.Title);
+            Assert.NotNull(_article.Text);
+        }
```

Comment:
This test is needless to with the same reason. You need to test other constructors like that:
```suggestion
        public void Ctor_TitleText_NotNull()
        {
            //Arrange
            string title = "Test article";
            //Act
            Article _article = new Article(title);
            //Assert
            Assert.NotNull(_article.Title);
            Assert.Equal(title, _article.Title);
        }
```